src/common/subgraphMapper.ts

```diff
@@ -151,7 +151,19 @@ export const mapOrdersArrayToInterface = (
     throw error;
   }
 };
-
+export const mapOrderArrayToPriceid = (response:any) =>{
+  if (response === null) return undefined;
```

Comment:
is better evaluate ```if(!response)``` because we remove more each cases like possible undefined and things like that